TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -211,6 +211,45 @@ function LocateHighestVersionSqlPackageWithSql()
 
 function LocateHighestVersionSqlPackageWithDacMsi()
 {
+    $sqlDataTierFrameworkRegKeyWow = "HKLM:", "SOFTWARE", "Wow6432Node", "Microsoft", "Microsoft SQL Server", "Data-Tier Application Framework" -join [System.IO.Path]::DirectorySeparatorChar
+    $sqlDataTierFrameworkRegKey = "HKLM:", "SOFTWARE", "Microsoft", "Microsoft SQL Server", "Data-Tier Application Framework" -join [System.IO.Path]::DirectorySeparatorChar
+
+    if (-not (Test-Path $sqlDataTierFrameworkRegKeyWow))
+    {
+        $sqlDataTierFrameworkRegKeyWow = $sqlDataTierFrameworkRegKey
```

Comment:
declare a new variable and use it after this if block. Otherwise it's confusing in the next code